DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -87,67 +87,61 @@ private IEnumerable<CourseStatistics> GetSampleCourses()
         }
 
         [Test]
-        public void GetAllCoursesForDelegate_should_call_correct_data_service_and_helper_methods()
+        public void GetDelegateAttemptsAndCourseCustomPrompts_should_call_correct_data_service_and_helper_methods()
         {
             // Given
             const int delegateId = 20;
             const int customisationId = 111;
-            var attemptStats = (7, 4);
+            var returnedAttemptStats = (10, 5);

```

Comment:
"returned" is quite vague about what's returning it. You could clarify it as `attemptStatsReturnedByDataService`.